.github/workflows/pull_request.yml

```diff
@@ -27,3 +27,8 @@ jobs:
           just build
           just up
           just test
+      - uses: actions/upload-artifact@v4
+        if: ${{ !cancelled() }}
+        with:
+          name: playwright-traces
+          path: build/test-results/
```

Comment:
Uploads traces you can view at https://trace.playwright.dev/ if tests fail

Example: https://github.com/OrcaCollective/OpenOversight/actions/runs/13615275905/job/38057562364?pr=537#step:5:23

Reply:
This is _exceptionally_ cool - do you mind documenting it somewhere in the repo?

Reply:
Added documentation!